symfony/psr7-meta/1.0/manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-    "aliases": ["psr7", "psr-7", "psr17", "psr-17"],
+    "aliases": ["psr-7", "psr-17"],
```

Comment:
Why not keep all current aliases?

Reply:
He alias-generating logic already deals with dashes